app/assets/stylesheets/application.scss

```diff
@@ -10,6 +10,9 @@
  * defined in the other CSS/SCSS files in this directory. It is generally better to create a new
  * file per style scope.
  *
+ *= require_tree .
```

Comment:
These are redundant and may break precompile.  To use bootstrap properly, we need to use the `@import's below instead of require.  Try `@import "devise_bootstrap_views";` and removing all the require statements.  It should work and make it easier to customize as needed.